Domains/4-Application/Connectors/CivilInfrastructureFramework/CifCommon.ecschema.xml

```diff
@@ -1059,11 +1059,11 @@
             <Class class="ProfileParabolaAspect"/>
         </Target>
     </ECRelationshipClass>
-    <ECEntityClass typeName="ProfilePointbyIntersectionRuleAspect" displayLabel="Profile Intersection Point">
+    <ECEntityClass typeName="ProfilePointbyIntersectionRuleAspect">
         <BaseClass>CivilPresentation</BaseClass>
         <ECProperty propertyName="Station" typeName="double" displayLabel="Station" category="ProfilePointbyIntersectionRuleAttributes_Profile_Intersection_Point" priority="299800" kindOfQuantity="cifu:STATION"/>
         <ECProperty propertyName="Elevation" typeName="double" displayLabel="Elevation" category="ProfilePointbyIntersectionRuleAttributes_Profile_Intersection_Point" priority="299700" kindOfQuantity="cifu:LENGTH"/>
-        <ECProperty propertyName="Name" typeName="double" displayLabel="Name" category="ProfilePointbyIntersectionRuleAttributes_Profile_Intersection_Point" priority="299600" kindOfQuantity="cifu:LENGTH"/>
+        <ECProperty propertyName="Name" typeName="string" displayLabel="Name" category="ProfilePointbyIntersectionRuleAttributes_Profile_Intersection_Point" priority="299600" kindOfQuantity="cifu:LENGTH"/>
```

Comment:
changing the type of a property is a breaking change and it doesn't make much sense to put a KOQ of length on a string property

Reply:
Forgot to remove the KOQ. Shall this be deprecated and a new property with the same displaylabel will help pass checks?

Reply:
Deprecated 'Name' property and added a new property 'PntName' and will add mapping